views/mixins/overviewBody.jade

```diff
@@ -0,0 +1,46 @@
+mixin renderOverviewFields(obj, hasTitle, hasSubtitle)
+	section.group
+		div.panel
+			if hasTitle
+				div.panel-heading=obj.label
+			div.panel-body
+				if hasSubtitle
+					h4=obj.label
+				-	if (obj.body)
+					div=obj.body
+				-	if (obj.fields && Array.isArray(obj.fields))
+					ul.list-group
+						each field in obj.fields
+							li.list-group-item
+								span
+									strong=field.label + ':'
+								span='	' + field.value
+
+
+mixin renderTabs(tabs)
+	div.tabs
+		div
+			ul.nav.nav-tabs.nav-tabs-overview
+				each tab, index in tabs
+					- var tabLink = (tab.label || '').replace(/\s+/g, '-').toLowerCase() + index;
```

Comment:
This stuff shouldn't be within the view itself. Please surface all this code to a route that prepares everything properly for the view.